addons/l10n_sg_ubl_pint/models/account_tax.py

```diff
@@ -0,0 +1,26 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    # The classification is too precise to be able to do any kind of guessing, so we need it set.
+    # See https://docs.peppol.eu/poac/sg/2024-Q2/pint-sg/bis/#_goods_and_services_tax_gst
+    ubl_cii_tax_category_code = fields.Selection(
+        selection_add=[
+            ('SR', "SG - Local supply of goods and services"),
+            ('SRCA-S', "SG - Customer accounting supply made by the supplier"),
+            ('SRCA-C', "SG - Customer accounting supply made by the customer on supplier’s behalf"),
+            ('SROVR-RS', "SG - Supply of remote services accountable by the electronic marketplace under the Overseas Vendor Registration Regime"),
+            ('SROVR-LVG', "SG - Supply of low-value goods accountable by the redeliverer or electronic marketplace on behalf of third-party suppliers"),
+            ('SRLVG', "SG - Own supply of low-value goods"),
+            ('ZR', "SG - Supplies involving goods for export/ provision of international services"),
+            ('ES33', "SG - Specific categories of exempt supplies listed under regulation 33 of the GST (General) Regulations"),
+            ('ESN33', "SG - Exempt supplies other than those listed under regulation 33 of the GST (General) Regulations"),
```

Comment:
for this kind of exemption, don't we need to set an exemption reason as well ? Currently, the field `ubl_cii_tax_exemption_reason_code` will not be displayed on the tax, see the [view](https://github.com/odoo/odoo/blob/53d9b10bc9717b7653998fcc65e64e26b52716c1/addons/account_edi_ubl_cii_tax_extension/views/account_tax_views.xml#L11).
IMHO, this should depend on `_requires_exemption_reason`, so any loca can inherit `_requires_exemption_reason` without having to modify anything in the view

Reply:
In the doc format, the exemption isn't listed:

https://docs.peppol.eu/poac/sg/2024-Q2/pint-sg/trn-invoice/syntax/cac-TaxTotal-1/cac-TaxSubtotal/cac-TaxCategory/

There is also no list of codes that we could use for it. However, it's okay, IMO, since the type of exemption is listed in the tax category code.
If the exemption reason was a char field I would display it, but as it is not and we would need to add a computed field to get it I think we can do without for now.